src/components/data/DataTable.js

```diff
@@ -131,21 +132,41 @@ const DataTable = props => {
     setTotalRowCount(unfilteredCount)
   })
 
-  const deleteColumnUpdateEntities = attributeName => {
+  const getAllEntities = async () => {
+    const params = _.pickBy(_.trim, { pageSize: filteredCount, filterTerms: activeTextFilter })
+    const queryResults = await Ajax(signal).Workspaces.workspace(namespace, name).paginatedEntitiesOfType(entityType, params)
```

Comment:
I realize that this is an API limitation and that we do this same thing in several other places already, but I do want to call out that this `getAllEntities` call adds another path to hit an OOM vulnerability in Rawls which can cause production outages if the user is trying to clear the column for many rows. Analysis Journeys has done some work recently to make sure that this number is extremely high (I think 300k rows is the max per page) so I think this is fine at the moment.

In a similar vein to some of the other changes we've been discussing lately for supporting table-wide operations, I think we should look into modifying [this Rawls](https://rawls.dsde-prod.broadinstitute.org/#/entities/delete_entity_column) endpoint to accept a `retainColumnHeader=true` query param that would do this same work without having to load all of the entities